File: Examen Parcial/proyectostrategy/src/main/java/pe/edu/pucp/softinv/proyectostrategy/modelo/Conductor.java
package pe.edu.pucp.softinv.proyectostrategy.modelo;

public class Conductor {
    private Integer conductorId;
    private String paterno;
    private String materno;
    private String nombres;
    private String numLicencia;
    private TipoLicencia tipoLicencia;
    private Integer puntosAcumulados;

    public Integer getConductorId() {
        return conductorId;
    }

    public void setConductorId(Integer conductorId) {
        this.conductorId = conductorId;
    }

    public String getPaterno() {
        return paterno;
    }

    public void setPaterno(String paterno) {
        this.paterno = paterno;
    }

    public String getMaterno() {
        return materno;
    }

    public void setMaterno(String materno) {
        this.materno = materno;
    }

    public String getNombres() {
        return nombres;
    }

    public void setNombres(String nombres) {
        this.nombres = nombres;
    }

    public String getNumLicencia() {
        return numLicencia;
    }

    public void setNumLicencia(String numLicencia) {
        this.numLicencia = numLicencia;
    }

    public TipoLicencia getTipoLicencia() {
        return tipoLicencia;
    }

    public void setTipoLicencia(TipoLicencia tipoLicencia) {
        this.tipoLicencia = tipoLicencia;
    }

    public Integer getPuntosAcumulados() {
        return puntosAcumulados;
    }

    public void setPuntosAcumulados(Integer puntosAcumulados) {
        this.puntosAcumulados = puntosAcumulados;
    }
    
    
}

File: Examen Parcial/P3_20221473/TransitSoft/TransitSoftDomain/Conductor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TransitSoftDomain
{
    public class Conductor
    {
        public int ConductorId { get; set; }
        public string Paterno { get; set; }
        public string Materno { get; set; }
        public string Nombres { get; set; }
        public string NumLicencia { get; set; }
        public TipoLicencia TipoLicencia { get; set; }
        public int PuntosAcumulados { get; set; }

    

        public Conductor()
        {
       
            TipoLicencia = new TipoLicencia();
        }



        public Conductor(string paterno, string materno, string nombres, string numLicencia, TipoLicencia TipoLicencia)
        {

            Paterno = paterno;
            Materno = materno;
            Nombres = nombres;
            NumLicencia = numLicencia;
            this.TipoLicencia = TipoLicencia;
         
        }



        public override string ToString()
        {
            return $"{Paterno} {Materno} {Nombres} - Licencia: {NumLicencia}";
        }
    }
}
